### PR TITLE
Added new ios integration methods documentation.

### PR DESCRIPTION
**Pull request by pbiosca-uz**

### CODE REVIEW

File: ios/sdk-ios-faq.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ This is a list of common questions about Userzoom SDK. If you cannot find an ans
 - [Should the string passed to `initWithTag()` and `Show()` always be the same?](#should-the-string-passed-to-initwithtag-and-show-always-be-the-same)
 - [How do I create a podfile?](#how-do-i-create-a-podfile)
 - [How do I run a project with cocoapods?](#how-do-i-run-a-project-with-cocoapods)
+- [How fix cocoapods could not find compatible versions for pod?](#how-fix-cocoapods-could-not-find-compatible-versions-for-pod)
```
Comment (gahing-uz) [inline ios/sdk-ios-faq.md:28]: How do I fix cocoapods could not find compatible versions for pod?

File: ios/sdk-ios-faq.md [hunk 2]
```diff
@@ -115,5 +116,14 @@ To run the project follow this steps:
 For more detailed information about how to use cocoapods please take a look to their official docs (https://guides.cocoapods.org/using/using-cocoapods.html)
 
 
+## How fix CocoaPods could not find compatible versions for pod?
```
Comment (gahing-uz) [inline ios/sdk-ios-faq.md:119]: How to fix CocoaPods could not find compatible versions for pod

File: ios/sdk-ios-faq.md [hunk 3]
```diff
@@ -115,5 +116,14 @@ To run the project follow this steps:
...
+
+If when you run the command `pod install`, you are getting this fail: CocoaPods could not find compatible versions for pod "DatadogSDKObjC
```
Comment (gahing-uz) [inline ios/sdk-ios-faq.md:121]: If when you run the command `pod install`, you are getting this error: CocoaPods could not find compatible versions for pod "DatadogSDKObjC

File: ios/sdk-ios-setup.md [hunk 4]
```diff
@@ -4,6 +4,14 @@
 
 >**Note: This SDK requires iOS 13.0 or greater.**
 
+## Adding the SDK via Swift Package Manager
+
+To integrate UserzoomSDK into your Xcode project using Swift Package Manager, you need add the repository url UserzoomSDK in your package.swift file or in Xcode, select File ▸ Swift Packages ▸ Add Package Dependency… :
```
Comment (gahing-uz) [inline ios/sdk-ios-setup.md:9]: To integrate the UserzoomSDK into your Xcode project using Swift Package Manager, you need to add the repository url UserzoomSDK in your package.swift file or in Xcode, select File ▸ Swift Packages ▸ Add Package Dependency… :

**Review by gahing-uz — CHANGES_REQUESTED** [review]
Check the in line comments to see the requested changes.

**Review by msobiepanek-uz — APPROVED** [review]

**Review by gahing-uz — APPROVED** [review]

**Pull request merged**